runtime/src/accounts_hash.rs

```diff
@@ -18,6 +22,28 @@ pub struct PreviousPass {
     pub lamports: u64,
 }
 
+/// parameters to calculate accounts hash
+pub struct AccountsHashConfig<'a> {
```

Comment:
yes, I imagine it will be passed through a few functions. Just trying to keep the name shorter. Maybe `CalcAccountsHashConfig` satisfies it all.

Reply:
#1 - I'm trying to move things out of accounts_db.rs over time since it is so large. This config is related to accounts hashing, so I was thinking I'd put it here.